### PR TITLE
Register special routes on deploy

### PR DESCRIPTION
**Pull request by heathd**
we've introduced a new `special_route` format[1] to allow us to register
unusual routes which don't have a real format associated with them.

This adds a rake task to register the HMRC contact route via that mechanism.

The intention is to run this rake task on deploy (as we do for the route
registration at present).

[1] alphagov/govuk-content-schemas#106

### CODE REVIEW

File: lib/tasks/publishing_api.rake [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+require 'gds_api/publishing_api'
+require 'gds_api/publishing_api/special_route_publisher'
+
+namespace :publishing_api do
+  desc 'Publish special routes via publishing api'
+  task publish_special_routes: :environment do
+    special_route_publisher = GdsApi::PublishingApi::SpecialRoutePublisher.new(logger: Logger.new(STDOUT))
+    special_route_publisher.publish(
+      title: 'HMRC contacts finder',
+      description: 'A filterable list of contact information from HMRC'
```
Comment (alext) [inline lib/tasks/publishing_api.rake]: Missing a trailing `,`

File: lib/tasks/publishing_api.rake [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      content_id: 'b110c03c-3f8d-4327-906b-17ebd872e6a6',
+      base_path: '/government/organisations/hm-revenue-customs/contact',
+      type: 'exact',
+      publishing_app: 'contacts-admin'
```
Comment (alext) [inline lib/tasks/publishing_api.rake]: and here.

File: lib/tasks/publishing_api.rake (hunk 1 above)
Comment (heathd) [inline lib/tasks/publishing_api.rake]: oops :-( thanks!

**Comment by alext** [comment]
:+1: Looks good to me.  Will need changes to alphagov-deployment though.

**Pull request merged**